lib/Constants.ts

```diff
@@ -0,0 +1,693 @@
+import { ConfigManager } from '@zwave-js/config'
+
+interface IGenericMap {
+	[key: number]: string
+}
+
+interface IHassClass {
+	[type: string]: {
+		[type: string]: string
+	}
+}
+
+interface ISensorProps {
+	device_class?: string
+	icon?: string
+}
+
+interface ISensorMap {
+	[type: string]: IGenericMap & {
+		props?: ISensorProps
+	}
+}
+
+interface IDeviceClassProps {
+	generic: string
+	specific: IGenericMap
+}
+
+interface IDeviceClass {
+	[genericClass: number]: IDeviceClassProps
+}
```

Comment:
This is just a `Record<number, IDeviceClassProps>`

Reply:
Isn't this the same?

Reply:
Yeah, but unnecessary to make an interface definition for something that's basically built in. If you want to alias the type, I'd do
```
type IDeviceClass = Record<number, IDeviceClassProps>
```

Reply:
But in that way how could I alias the key?